Sources/Certificate Pinning/CertificateHasher.swift

```diff
@@ -53,10 +53,10 @@ struct CertificateHasher {
     }
     
     static func checkValidity(of trust: SecTrust) -> Bool {
-        var result = SecTrustResultType.invalid
-        let status = SecTrustEvaluate(trust, &result)
+        var result: CFError?
+        let status = SecTrustEvaluateWithError(trust, &result)
         
-        return (status == errSecSuccess) ? (result == .unspecified || result == .proceed) : false
+        return status || result == nil
```

Comment:
I realize I'm a bit late to the party, but is there a reason why we adjusted the certificate pinning logic in this PR?

Reply:
`SecTrustEvalute()` has been deprecated in favor of `SecTrustEvaluateWithError(_:_:) `